MN/Algoritmi/LU/Crout/factorCrout.m

```diff
@@ -0,0 +1,13 @@
+function [L U] = factorCrout(A)
+	n = length(A);
+	L = zeros(n,n);
+	U = diag(ones(n,1));
+	for p = 1: n
```

Comment:
O linie goala inainte de for?
Separam initializarile de cod.
Also, spatiu dupa "1".